.github/workflows/create_trello_card_template.yaml

```diff
@@ -0,0 +1,26 @@
+name: Create Trello card
+
+on:
+  workflow_call:
+    secrets:
+      TRELLO_API_KEY:
+        required: true
+      TRELLO_API_TOKEN:
+        required: true
+
+jobs:
+  create-card:
+    runs-on: ubuntu-latest
+    steps:
+      - name: echo ref_name
+        run: echo GITHUB.HEAD_REF_NAME ${GITHUB_HEAD_REF_NAME}
+      - name: create a card
+        uses: nentgroup/pr-trello-action@v2
+        with:
+          api-key: '${{ secrets.TRELLO_API_KEY }}'
+          api-token: '${{ secrets.TRELLO_API_TOKEN }}'
+          list-id: '63ce5064a4859b0387f46b9b'
+          title-format: 'Dashboard PR: ${title}'
```

Comment:
How is the title set? Is that taken from the PR title?

Reply:
Yes, the trigger passes a bunch of info (source branch ref, target branch ref, etc) which includes the PR title.